src/config/cfg_rules.ini

```diff
@@ -11,7 +11,7 @@ section = ifp
 section = secrets
 section = kcm
 section_re = ^secrets/users/[0-9]\+$
-section_re = ^domain/.*$
+section_re = ^domain/[0-9A-Za-z.]\+$
```

Comment:
Looking at https://technet.microsoft.com/en-us/library/cc959336.aspx I think it was a mistake to implement any restrictions on domain names. Moreover the domain name in the section name is just a label that does not necessary needs to be the same as the actual DNS name. Also DNS itself does not impose any restrictions on the name labels (with the exception of length), however implementations vary in this.
I will remove the restrictions and close the related issue as wontfix.